src/nim_status.nim

```diff
@@ -2,7 +2,9 @@ from nim_status/types import
   SignalCallback
 
 from nim_status/lib/shim as nim_shim import
-  hashMessage
+  hashMessage,
+  generateAlias,
+  saveAccountAndLogin
```

Comment:
As mentioned during the call earlier this week, I think for now it's best to not expose `saveAccountAndLogin` at the top-level as not all the pieces are in place for it to work as a drop-in replacement for status-go's functionality.